pkg/web/static/js/envelope.js

```diff
@@ -96,6 +96,19 @@ document.body.addEventListener('htmx:afterSettle', (e) => {
       const readableIdentifierType = IDENTIFIER_TYPE[identifierCode];
       identifierType.textContent = readableIdentifierType || identifierCode;
     });
-  }
-})
+  };
 
+  disableSubmitButton();
+}
+);
+
+// Disable submit button to prevent multiple form submissions.
```

Comment:
The send envelope page refreshes once a form has been submitted. That is why there's only a disable function for this form.